pkg/liqoctl/status/handler.go

```diff
@@ -16,17 +16,54 @@ package status
 
 import (
 	"context"
+	"errors"
+
+	"github.com/pterm/pterm"
 
 	"github.com/liqotech/liqo/pkg/liqoctl/factory"
 )
 
+// CollectionError struct holding the error for a given deployment of Liqo while collecting its status.
+type CollectionError struct {
+	AppType string
+	AppName string
```

Comment:
You should probably generalize the field names. I would also evaluate whether it makes sense to keep them or rather it is better to only enrich the error. If kept as now, it would be probably appropriate to create a String() function to consistently convert this struct into a string.